pkg/harvester/edit/network.harvesterhci.io.vlanconfig/index.vue

```diff
@@ -361,21 +361,6 @@ export default {
 
       this.nicErrors = uniq(nicErrors);
     },
-
-    'value.spec.uplink.nics'(nics = []) {
```

Comment:
We don't need to show warning banner after selected nic. Since disabled options cannot be selected.